source/tests/utils_test.h

```diff
@@ -62,15 +62,24 @@ inline transit_realtime::TripUpdate
 make_delay_message(const std::string& vj_uri,
```

Comment:
Same thing, this is not my creation, Maybe in an other PR with title "improving realtime test readability" ?
It is just because the PR to already heavy

Reply:
JIRA : https://jira.kisio.org/browse/NAVP-1228